lib/dor/text_extraction/ocr.rb

```diff
@@ -11,6 +11,11 @@ def initialize(params = {})
         @workflow_context = params[:workflow_context]
       end
 
+      def abbyy_output_path
+        bare_druid = @cocina_object.externalIdentifier.split(':')[-1]
```

Comment:
1. The attr_reader on line 7 means we can don't need to use the instance variable here and would be more consistent with how's it's done in other parts of the codebases
2. We do this to get the bare_druid in other parts of the codebase, so I'd do it this way, again for consistency:

```
 bare_druid = cocina_object.externalIdentifier.delete_prefix('druid:')
```

Reply:
Updated. I was going with (sort of) consistency for the current workcycle with split so we should probably update these methods.

https://github.com/sul-dlss/common-accessioning/blob/bd7c12805820dbe6c5b75726a2c44569a651a3a6/lib/robots/dor_repo/ocr/fetch_files.rb#L30

https://github.com/sul-dlss/common-accessioning/blob/bd7c12805820dbe6c5b75726a2c44569a651a3a6/lib/dor/text_extraction/abbyy/ticket.rb#L26

Reply:
Ugh, we do this all over the place in so many different ways.  It's not that big a deal, just an annoyance to keep repeating ourselves in different ways.